pydatastructs/miscellaneous_data_structures/stack.py

```diff
@@ -96,7 +96,7 @@ def __new__(cls, maxsize=None, top=0, items=None, dtype=int):
         if items._size > maxsize:
             raise ValueError("Overflow, size of items %s is greater "
                             "than maxsize, %s"%(items._size, maxsize))
-        obj = object.__new__(cls)
+        obj = Stack.__new__(cls)
```

Comment:
We have to keep it as `object`. This is an exception to the generic fix. Rest is fine.

Reply:
Done!!